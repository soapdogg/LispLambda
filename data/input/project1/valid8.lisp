(LET
	((X 1)
		(Y (JIM X 2)))
(ERIC X Y))