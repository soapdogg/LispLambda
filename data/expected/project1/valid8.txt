LITERAL ATOMS: 8, LET, X, Y, JIM, X, ERIC, X, Y
NUMERIC ATOMS: 2,3
OPEN PARENTHESES: 6
CLOSING PARENTHESES: 6